packages/nextjs/src/components/_common/uxsignalsWidget/UxSignalsWidget.tsx

```diff
@@ -19,7 +20,7 @@ type Consent = {
 const uxSignalsScriptUrl = 'https://widget.uxsignals.com/embed.js';
 let scriptAddTimeout: ReturnType<typeof setTimeout>;
 
-export const UxSignalsWidget = ({ embedCode }: UxSignalsWidgetProps) => {
+export const UxSignalsWidgetComponent = ({ embedCode }: UxSignalsWidgetProps) => {
```

Comment:
Denne trengs kanskje ikke eksporteres lenger
```suggestion
const UxSignalsWidgetComponent = ({ embedCode }: UxSignalsWidgetProps) => {
```